tools/internal_ci/linux/grpc_bazel_distribtest.cfg

```diff
@@ -0,0 +1,26 @@
+# Copyright 2017 gRPC authors.
```

Comment:
Generally I think we should avoid creating more kokoro jobs as these become hard to maintain, but for now I don't have a better alternative, so creating the 2 new kokoro jobs seems acceptable.